pkg/reconciler/vertex/scaling/scaling.go

```diff
@@ -140,11 +141,12 @@ func (s *Scaler) scaleOneVertex(ctx context.Context, key string, worker int) err
 	log := logging.FromContext(ctx).With("worker", fmt.Sprint(worker)).With("vertexKey", key)
 	log.Debugf("Working on key: %s", key)
 	strs := strings.Split(key, "/")
-	if len(strs) != 2 {
+	if len(strs) != 3 {
```

Comment:
What is the reason we need to touch autoscaling?

Reply:
key I believe is the api endpoint in string format ('namespace/vertex/pods') which is split on '/'. Since I added pod count to the endpoint, I had to modify the function signature by accepting an additional parameter.